pytest/lib/bridge.py

```diff
@@ -94,15 +222,19 @@ def start(self):
                 '--port',
```

Comment:
Remove both TODO:

```
# TODO use blockTime
# TODO set params
```

Reply:
We still don't use custom params to start ganache. We also don't handle stderr/stdout for cases if we want to simulate Eth forks.
I will remove only `# TODO use blockTime` as we use `ganache_block_prod_time` param from config.